homeassistant/helpers/selector.py

```diff
@@ -162,3 +162,17 @@ class ActionSelector(Selector):
     """Selector of an action sequence (script syntax)."""
 
     CONFIG_SCHEMA = vol.Schema({})
+
+
+@SELECTORS.register("object")
+class ObjectSelector(Selector):
+    """Selector for an arbitrary object."""
+
+    CONFIG_SCHEMA = vol.Schema({})
+
+
+@SELECTORS.register("text")
```

Comment:
It's weird that no selector is single line text and `text` can be multi-line.

Reply:
should there be an attribute `multiline: true` ? 🤷

Reply:
Any ideas for a better name?

Reply:
Omitting the `selector` entirely will give a string input. So `multiline: false` (which I assume would be default) would just be the same as nothing at all 🤔

Reply:
But the implementation would be easy enough, and it'd be easy to understand.

Reply:
It would be good if having no selector also results in a selector being picked. In this case being text with no extra options. If multiline is false by default, that would work.